internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -215,6 +215,43 @@ paths:
               schema:
                 type: string
 
+  '/composes/{id}/sboms':
+    get:
+      operationId: getComposeSBOMs
+      summary: Get the SBOMs for a compose.
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+            format: uuid
+            example: 123e4567-e89b-12d3-a456-426655440000
+          required: true
+          description: ID of compose SBOMs to get
```

Comment:
NITPICK: This is a bit awkwardly phrased and might be misunderstood as "ID of SBOMs".  In other words, it kinda sounds like "compose-SBOM" is the object being fetched, so it wants the ID of that, but instead it's the compose ID it needs.

Maybe something like:
> ID of compose for which to get SBOMs

though I'm not too happy about that one either.

Reply:
I agree that this is not great by any means :smile: I'll happily use your suggestion.